sdk/lib/opentelemetry/sdk.rb

```diff
@@ -61,6 +61,11 @@ module SDK
     #       c.use_all
     #     end
     def configure
+      if ENV['OTEL_SDK_DISABLED'] == 'true'
```

Comment:
I see that you had added but then removed tests. 

Where the tests failing?

Reply:
I created the test incorrectly

Reply:
Test recreated now